js/msp/MSPCodes.js

```diff
@@ -11,8 +11,8 @@ var MSPCodes = {
     MSP_NAME:                   10,
     MSP_SET_NAME:               11,
 
-    MSP_CHANNEL_FORWARDING:     32,
-    MSP_SET_CHANNEL_FORWARDING: 33,
+    MSP_CHANNEL_FORWARDING:     32, // Not used
```

Comment:
I wouldn't update too much in here, we'll be generating this file form the C header file in the future.